## /bin/sh -e simulate dpatch for prepend_chlog.py
## Build
## nomarch.dpatch by Kilian Krause <kilian@debian.org>
## DP: Don't optimize for i686 anywhere
## -- upstream will move to configure with 1.4

Index: asterisk-1.2.27/Makefile
===================================================================
--- asterisk-1.2.27.orig/Makefile	2008-03-19 11:33:09.000000000 +0100
+++ asterisk-1.2.27/Makefile	2008-03-19 11:33:20.000000000 +0100
@@ -237,10 +237,6 @@
   ASTCFLAGS+=-I$(CROSS_COMPILE_TARGET)/usr/local/include -L$(CROSS_COMPILE_TARGET)/usr/local/lib
 endif
 
-ifneq ($(PROC),ultrasparc)
-  ASTCFLAGS+=$(shell if $(CC) -march=$(PROC) -S -o /dev/null -xc /dev/null >/dev/null 2>&1; then echo "-march=$(PROC)"; fi)
-endif
-
 ifeq ($(PROC),ppc)
   ASTCFLAGS+=-fsigned-char
 endif
